code/edgebridge/src/test/java/com/adobe/marketing/mobile/edge/bridge/EdgeBridgeExtensionTests.java

```diff
@@ -42,67 +38,102 @@
 import org.junit.runner.RunWith;
 import org.mockito.ArgumentCaptor;
 import org.mockito.Mock;
+import org.mockito.MockedStatic;
 import org.mockito.Mockito;
-import org.powermock.api.mockito.PowerMockito;
-import org.powermock.core.classloader.annotations.PrepareForTest;
-import org.powermock.modules.junit4.PowerMockRunner;
+import org.mockito.junit.MockitoJUnitRunner;
 
-@RunWith(PowerMockRunner.class)
-@PrepareForTest({ MobileCore.class, ExtensionApi.class, ExtensionErrorCallback.class, ExtensionError.class })
+@RunWith(MockitoJUnitRunner.class)
 public class EdgeBridgeExtensionTests {
 
 	private EdgeBridgeExtension extension;
 
 	@Mock
 	ExtensionApi mockExtensionApi;
 
-	@Mock
-	Application mockApplication;
-
-	@Mock
-	Context mockContext;
-
+	// Performs pre-test setup before each test case
+	// Resets the mocked ExtensionApi instance mockExtensionApi (handles event dispatch)
+	// Creates a new instance of the EdgeBridgeExtension using the ExtensionApi mock
 	@Before
 	public void setup() {
-		PowerMockito.mockStatic(MobileCore.class);
-		Mockito.when(MobileCore.getApplication()).thenReturn(mockApplication);
-		Mockito.when(mockApplication.getApplicationContext()).thenReturn(mockContext);
-
+		Mockito.reset(mockExtensionApi);
 		extension = new EdgeBridgeExtension(mockExtensionApi);
 	}
 
 	// ========================================================================================
 	// constructor
 	// ========================================================================================
+	@Test
+	public void test_registerExtension_deprecated() {
```

Comment:
- how are we testing the new method of registration?
- can we move these tests to EdgeBridgeTests since they test the testing the public APIs and not the internal EdgeBridgeExtension?